### PR TITLE
zipl: be verbose about secure boot state

### PR DESCRIPTION
**Pull request by sharkcz**
When trying to understand what's going on in https://bugzilla.redhat.com/show_bug.cgi?id=2089134 (IBM LTC bz 198327) where the secure boot boot record is not always written, I found it would be good have zipl to report the state of secure boot support on the system.

### CODE REVIEW

File: zipl/src/bootmap.c [hunk 1]
```diff
@@ -125,7 +128,7 @@ check_secure_boot_support(void)
 	FILE *fp;
 
 	if (verbose)
-		printf("  Secure boot support: ");
+		printf("Secure boot support: ");
```
Comment (stefan-haberland) [inline zipl/src/bootmap.c:131]: This should be part of the first patch.

**Comment by stefan-haberland** [comment]
Overall this looks good to me. Only one thing that should be improved. Thanks.

File: zipl/src/bootmap.c (hunk 1 above)
Comment (sharkcz) [reply zipl/src/bootmap.c:131]: OK

**Comment by sharkcz** [comment]
updated and rebased

**Comment by sharkcz** [comment]
Hi @hoeppnerj , could you take a look, please?

**Comment by hoeppnerj** [comment]
Pulled and merged. Thanks a lot @sharkcz !

**Pull request closed without merging**